scripts/test

```diff
@@ -5,5 +5,4 @@ source "${BASH_SOURCE%/*}/../node_modules/sanctuary-scripts/functions"
 
 branches="$(get min-branch-coverage)"
 
-node_modules/.bin/nyc --extension .mjs node_modules/.bin/mocha
-node_modules/.bin/nyc check-coverage --branches $branches
```

Comment:
I haven't been able to find an alternative coverage reported that works on mjs files without a compile step.